contracts/interfaces/IWrappedExternalBribeFactory.sol

```diff
@@ -0,0 +1,5 @@
+pragma solidity 0.8.13;
```

Comment:
[Note] This interface will be used in `createGauge` function in `Voter` contract to create bribe.